chia/wallet/wallet_state_manager.py

```diff
@@ -1234,10 +1439,10 @@ async def _add_coin_states(
                         else:
                             await self.coin_store.set_spent(coin_name, uint32(coin_state.spent_height))
                             rem_tx_records: List[TransactionRecord] = []
-                            for out_tx_record in all_unconfirmed:
```

Comment:
This is some unrelated renaming in VC code, not sure why its in here :)

Reply:
Was changed something here but reverted, fixed